packages/mc-html-template/lib/load-env.js

```diff
@@ -33,13 +31,11 @@ const substituteEnvVariablePlaceholder = (
     );
   }
 
-  const substitutedEnvConfig = replaceall(
-    matchedString,
-    valueOfEnv,
-    valueOfEnvConfig
+  const escapedMatchedString = matchedString.replace(/[${}:]/g, '\\$&');
```

Comment:
Haha. This is what I liked having the lib for. All cool :)

Reply:
Just because you hate regular expressions 😜

Reply:
I propose we rewrite the MC in Regular Expressions.